experimenter/tests/nimbus_integration_tests.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -euo pipefail
+set +x
+
+export PATH=$PATH:/home/seluser/.local/bin
+
+if [[ -n "${UPDATE_FIREFOX_VERSION}" ]]; then
+    sudo ./experimenter/tests/integration/nimbus/utils/nightly-install.sh
+fi
+
+sudo apt-get -qqy update && sudo apt-get -qqy install python3-venv python3-pip
+pip install poetry --break-system-packages
```

Comment:
Neat, i've neve seen `--break-system-packages` before. I think that maybe warrants a comment